src/status_im/communities/core.cljs

```diff
@@ -110,15 +111,29 @@
 (rf/defn left
   {:events [::left]}
   [cofx response-js]
-  (rf/merge cofx
-            (handle-response cofx response-js)
-            (navigation/pop-to-root-tab :chat-stack)
-            (activity-center/notifications-fetch-unread-count)))
+  (let [community-name (aget response-js "communities" 0 "name")]
+    (rf/merge cofx
+              {:dispatch-n [[:sanitize-messages-and-process-response response-js]
```

Comment:
better to do not use `:dispatch-n`, we should keep `handle-response` and also use handler for `toasts/create` as well